config/sets/sylius/plus/up-to-plus-modular.php

```diff
@@ -10,22 +10,23 @@
 
 return static function(RectorConfig $rectorConfig): void {
     $rectorConfig->ruleWithConfiguration(RenameClassRector::class, [
+        'Sylius\Plus\Controller\DashboardController' => 'Sylius\MultiStorePlugin\ChannelAdmin\Infrastructure\Controller\DashboardController',
+        'Sylius\Plus\Factory\VariantsQuantityMapFactory' => 'Sylius\MultiSourceInventoryPlugin\Integration\PartialShipment\Factory\VariantsQuantityMapFactory',
+        'Sylius\Plus\Factory\VariantsQuantityMapFactoryInterface' => 'Sylius\MultiSourceInventoryPlugin\Integration\PartialShipment\Factory\VariantsQuantityMapFactoryInterface',
         'Sylius\Plus\Inventory\Infrastructure\Ui\SplitShipmentAction' => 'Sylius\MultiSourceInventoryPlugin\Integration\PartialShipment\Action\SplitShipmentAction',
         'Sylius\Plus\Inventory\Infrastructure\Validator\StockSufficientForInventorySource' => 'Sylius\MultiSourceInventoryPlugin\Integration\PartialShipment\Validator\StockSufficientForInventorySource',
         'Sylius\Plus\Inventory\Infrastructure\Validator\StockSufficientForInventorySourceValidator' => 'Sylius\MultiSourceInventoryPlugin\Integration\PartialShipment\Validator\StockSufficientForInventorySourceValidator',
         'Sylius\Plus\Returns\Infrastructure\Ui\Admin\ReturnUnitsToInventoryAction' => 'Sylius\Plus\Ui\Admin\ReturnUnitsToInventoryAction',
         'Sylius\Plus\Returns\Infrastructure\Ui\Admin\SelectUnitsToReturnToInventoryAction' => 'Sylius\ReturnPlugin\Infrastructure\Ui\Admin\SelectUnitsToReturnToInventoryAction',
         'Sylius\Plus\SharedKernel\Exception\ResourceNotSupportedException' => 'Sylius\SharedKernel\Exception\ResourceNotSupportedException',
         'Sylius\Plus\SharedKernel\ResourceChannelCheckerInterface' => 'Sylius\SharedKernel\Contract\ResourceChannelCheckerInterface',
-        'Sylius\Plus\Controller\DashboardController' => 'Sylius\MultiStorePlugin\ChannelAdmin\Infrastructure\Controller\DashboardController',
-        'Sylius\Plus\Factory\VariantsQuantityMapFactory' => 'Sylius\MultiSourceInventoryPlugin\Integration\PartialShipment\Factory\VariantsQuantityMapFactory',
-        'Sylius\Plus\Factory\VariantsQuantityMapFactoryInterface' => 'Sylius\MultiSourceInventoryPlugin\Integration\PartialShipment\Factory\VariantsQuantityMapFactoryInterface',
     ]);
 
     $rectorConfig->ruleWithConfiguration(RenameNamespaceRector::class, [
         'Sylius\Plus\BusinessUnits' => 'Sylius\MultiStorePlugin\BusinessUnits',
         'Sylius\Plus\ChannelAdmin' => 'Sylius\MultiStorePlugin\ChannelAdmin',
         'Sylius\Plus\CustomerPools' => 'Sylius\MultiStorePlugin\CustomerPools',
+        'Sylius\Plus\Entity' => 'Sylius\MultiStorePlugin\ChannelAdmin\Domain\Model',
```

Comment:
I'm not sure if it is a proper rule, as there were more classes in this namespace and they were not transferred to `ChannelAdmin`, probably you should write the more specific one 🤔